packages/utils/src/signatures.ts

```diff
@@ -49,6 +50,12 @@ export async function isValidEip1271Signature(
   projectId: string,
   baseRpcUrl?: string,
 ) {
+  const parsedChain = parseChainId(chainId);
+  if (!parsedChain.namespace || !parsedChain.reference) {
+    throw new Error(
+      `isValidEip1271Signature failed: chainId must be in CAIP-2 format, received: ${chainId}`,
```

Comment:
Why should it be in CAIP-2 format since EIP-1271 signature is specific to EVM ?

Reply:
blockchain api rejects the request if it isnt. Probably we can update there it there.

Reply:
it shouldn't. These params are optionated for our rpc api. Direct node urls do not use them